helm/conjur-config-cluster-prep/templates/golden_configmap.yaml

```diff
@@ -20,14 +20,10 @@ data:
     authnK8sAuthenticatorID: {{ required "A valid authnK8s.authenticatorID is required!" .Values.authnK8s.authenticatorID }}
     {{- if eq .Values.authnK8s.clusterRole.create true }}
     authnK8sClusterRole: {{ .Values.authnK8s.clusterRole.name | default "conjur-clusterrole" }}
-    {{- else }}
```

Comment:
Instead of simply deleting this `{{- else }}` conditional, this should be replaced with an `else if` that looks at the `.Values.conjurConfigMap.authnMethod` setting. If `authnMethod` is not set for JWT, then the `authnK8sClusterRole` setting is still required.

Same for `authnK8sServiceAccount` on line 29 below.

This also means that the 2 UT test cases that were deleted are still needed.

Reply:
Ah, never mind on that ^^^. The `authnMethod` setting doesn't apply to the cluster prep Helm chart.
This is unfortunate, since it opens the door a bit for un-flagged user misconfiguration.